### PR TITLE
Migrate to ghcr.io

### PR DESCRIPTION
**Pull request by yokaze**
- Migrate to ghcr.io
- Confirm `make version` and `make update-kustomize-version` work properly

Signed-off-by: Daichi Sakaue <daichi-sakaue@cybozu.co.jp>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -189,13 +189,13 @@ version: login-gh update-kustomize-version ## Update dependent versions
 	$(call update-version,kubernetes-sigs/kind,KIND_VERSION)
 	$(call update-version,rust-lang/mdBook,MDBOOK_VERSION)
 
-	$(call update-version-quay,cert-manager,CERT_MANAGER_VERSION)
-	$(call update-version-quay,cilium,CILIUM_VERSION)
+	$(call update-version-ghcr,cert-manager,CERT_MANAGER_VERSION)
+	$(call update-version-ghcr,cilium,CILIUM_VERSION)
```
Comment (terassyi) [inline Makefile:193]: `cilium` container image doesn't exist in ghcr.io now.
So we have to wait for creating it or keep using quay.io until migrating cilium's image.

Comment (yokaze) [reply Makefile:193]: @terassyi 
Thanks for the reminder. Does it ok to proceed now?

Comment (terassyi) [reply Makefile:193]: Sorry for my late reply. 
It's ok to do.

**Review by terassyi — APPROVED** [review]
LGTM

**Pull request merged**